WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -365,6 +370,18 @@ extension AddAttributeOptionsViewController {
         present(actionSheet, animated: true)
     }
 
+    /// Navigates to `RenameAttributesViewController`
+    ///
+    private func navigateToRenameAttribute() {
+        guard let attributeName = self.viewModel.titleView else {
```

Comment:
nit: This is an unnecessary `self` since we are not inside a closure!

Reply:
I think there is no reason for `titleView` to be optional (I know it was like that before this PR 😛)
And, since we are using it for more than the title view, probably is best if we rename the property to `attributeName` or something like that, what do you think?